modules/apps/headless/headless-delivery/headless-delivery-test/src/testIntegration/java/com/liferay/headless/delivery/resource/v1_0/test/StructuredContentResourceTest.java

```diff
@@ -646,6 +650,61 @@ public void testPostStructuredContentFolderStructuredContent()
 				getMarkedAsDefault());
 	}
 
+	@Test
+	public void testPostStructuredContentWithBatch() throws Exception {
```

Comment:
I think Brian will reject this method as it does not override any other method. Which pattern should we follow here? @4lejandrito

Reply:
What we are doing in objects is to have [a specific file to test the batch endpoints](https://github.com/liferay/liferay-portal/blob/419728b802858bf821f2ba8b98be33780ff961ab/modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/headless/batch/engine/resource/v1_0/test/ExportTaskResourceTest.java). Maybe we can do the same here? But if we do, we should use the `/headless-batch-engine` API to be consistent.